### PR TITLE
add tolerations to common logging field

### PR DESCRIPTION
**Pull request by paynejacob**
For: https://github.com/rancher/rancher/issues/20565

Added tolerations to all logging configurations objects.

### CODE REVIEW

File: apis/management.cattle.io/v3/logging_types.go [hunk 1]
```diff
@@ -52,6 +52,7 @@ type LoggingCommonField struct {
 	OutputFlushInterval int               `json:"outputFlushInterval,omitempty" norman:"default=60"`
 	OutputTags          map[string]string `json:"outputTags,omitempty"`
 	EnableJSONParsing   bool              `json:"enableJSONParsing,omitempty"`
+	Tolerations			*[]v1.Toleration	  `json:"tolerations,omitempty"`
```
Comment (cbron) [inline apis/management.cattle.io/v3/logging_types.go]: * Make sure gofmt is enabled/run, you can start a watcher with Goland. 
* Any reason to make it a pointer ? Typically we do that to say null or not, but here its an array so we can just check if empty.

Comment (paynejacob) [reply apis/management.cattle.io/v3/logging_types.go]: I saw it as a pointer in other locations but for our use case an empty array works better.

**Comment by cbron** [comment]
lgtm, but can you squash your 3 commits into one

**Comment by paynejacob** [comment]
This functionality exists in logging v2 on rancher 2.5.0. This version of logging is deprecated.

**Pull request closed without merging**